person.rb

```diff
@@ -22,6 +24,10 @@ def correct_name
 
   private
 
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
```

Comment:
- [ ] Kindly move this function above the `private` line 25. as this is used as a public function.